setup.py

```diff
@@ -36,6 +36,7 @@ def read(*rnames):
         'Products.statusmessages',
         'Zope2',  # Globals, OFS(tests),
         'decorator',
+        'plone.app.contentlisting',
```

Comment:
Hm doesn't this introduce circular dependencies? Does Plone depend on p.a.cl or not?